components/odh-notebook-controller/e2e/validation_test.go

```diff
@@ -0,0 +1,103 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	apiext "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func testControllerValidation(t *testing.T) {
+	testCtx, err := NewTestContext()
+	require.NoError(t, err)
+	t.Run("notebook-controller Validation", testCtx.testNotebookController)
```

Comment:
Replace with the following code to explicitly indicate we are testing kubeflow code:

```
t.Run("Validate Kubeflow notebook controller", testCtx.testKubeflowNotebookController)
```